package.json

```diff
@@ -38,8 +38,8 @@
     "prod:release": "yarn prod && yarn prod:zip",
     "lint": "eslint  --ignore-path .gitignore packages",
     "lint:fix": "yarn run lint --fix",
-    "test:unit": "cross-env NODE_OPTIONS=--experimental-vm-modules NODE_ENV=test jest --no-cache --testPathIgnorePatterns '.*\\.integration\\.test\\.jsx?'",
-    "test:integration": "cross-env NODE_OPTIONS=--experimental-vm-modules NODE_ENV=test node ./scripts/integration-test.js --no-cache --testMatch '**/*.integration.test.[jt]s?(x)' --coverageDirectory './coverage/integration'",
+    "test:unit": "cross-env NODE_OPTIONS=--experimental-vm-modules NODE_ENV=test jest --no-cache --coverage --testPathIgnorePatterns '.*\\.integration\\.test\\.jsx?'",
+    "test:integration": "cross-env NODE_OPTIONS=--experimental-vm-modules NODE_ENV=test node ./scripts/integration-test.js --no-cache --coverage --config='./integration.jest.config.js'",
```

Comment:
don't add --coverage here
it will generate coverage report on every run if start with `yarn test:unit --watch`
it slows down local development most of the time

Reply:
done